package.json

```diff
@@ -8,17 +8,17 @@
     "dist"
   ],
   "scripts": {
-    "build": "npm run build:dev && npm run build:prod",
-    "build:dev": "webpack --progress",
-    "build:dev:local": "webpack --config-name local-webpack.config.js --progress",
-    "build:prod": "webpack --progress --mode=production; echo 'Copying databrowser.html to index.html for backwards compatibility; can be removed when Node Solid Server is updated.'; cp dist/databrowser.html dist/index.html",
-    "build-no-min": "npm run build:dev",
-    "prepublishOnly": "npm run build",
-    "start": "webpack-dev-server --https",
+    "build": "npm run clean && npm run build-version && npm run build-dev && npm run build-dist",
+    "build-dev": "webpack --progress",
+    "build-dist": "webpack --progress --mode=production; echo 'Copying databrowser.html to index.html for backwards compatibility; can be removed when Node Solid Server is updated.'; cp dist/databrowser.html dist/index.html",
+    "build-version": "./timestamp.sh > src/versionInfo.ts  && eslint 'src/versionInfo.ts' --fix",
+    "watch": "npm run version && webpack --watch",
```

Comment:
Shouldn't this be `npm run build-version`?

Reply:
Yup, fixed typo in another commit ^_^